src/server/game/Entities/Player/Player.cpp

```diff
@@ -6695,6 +6695,42 @@ void Player::RewardReputation(Quest const* quest)
     }
 }
 
+/*
+ *  If in a battleground or battlefield player dies, and an enemy removes the insignia, the player's bones is lootable
+ *  Called by remove insignia spell effect
+ */
+void Player::RemovePVPInsignia(Player* looter)
```

Comment:
Is this actually a thing for wintergrasp? Was definitely not true in tol barad and ashran

Reply:
see this "reliable" source https://www.mmo-champion.com/threads/768794-Do-you-remove-insignias
omegalul